asm6f.c

```diff
@@ -1871,6 +1881,13 @@ int main(int argc,char **argv) {
 				case 'f':
 					genlua=1;
 					break;
+				// [unregistered addition]
+				case 'U':
+					vacillatemacro=1;
```

Comment:
Is the fallthrough here intentional?

Reply:
Yes the fallthrough on case U is intentional.  (Just like loopy's -L -l fallthrough is intentional.)  That's where that fallthrough idea came from; furthermore, the fallthrough on -U reduced the boolean algebra result on 1687.  It is simpler and takes less lines of code. :) Both beneficial, for me at least. :)

Reply:
Yes that's intentional. :) Both -u and -U set needrept to 0 and set listfilename to true_ptr (if listfilename==true_ptr then asm6 creates a listing file... listfilename=true_ptr is also underneath -L and -l; after think about this for quite a while, it doesn't seem possible to conveniently have -L, -l, -U, and -u to use the same listfilename=true_ptr).  However, -U also sets vacillatemacro to 1.  The state of the flags needrept and vacillatemacro help the program to know how to branch after loading them in other places in this commit's code.  I have a page of boolean algebra that verifies what you encounter when you run this commit.  It does work freem.

The simplicity is purposeful.  Personally, I didn't want to cause the user's computer to have to do an extensive amount of work compared to asm6.